blockchain/blockchain_test.go

```diff
@@ -1644,3 +1659,400 @@ func TestBlockchain_WriteFullBlock(t *testing.T) {
 	require.NotNil(t, db[hex.EncodeToHex(getKey(storage.CANONICAL, common.EncodeUint64ToBytes(header.Number)))])
 	require.NotNil(t, db[hex.EncodeToHex(getKey(storage.RECEIPTS, header.Hash.Bytes()))])
 }
+
+func TestDiskUsageWriteBatchAndUpdate(t *testing.T) {
+	const (
+		checkInterval  = 10000 //milliseconds
```

Comment:
Express time in `time.Duration` since it is more readable and friendly.

```suggestion
		checkInterval  = 10*time.Second
		numberOfBlocks = 100
		blockTime = 2*time.Second
```